website/source/index.html.erb

```diff
@@ -1,144 +1,434 @@
+        <h3>3. Run Applications</h3>
+        <p>
+          Nomad runs applications and ensures they keep running in failure
+          scenarios. In addition to long-running services, Nomad can schedule
+          batch jobs, distributed cron jobs, or parameterized jobs.
+        </p>
+      </div>
+      <div class="col-sm-6">
+        <h3>4. Monitor Progress</h3>
+        <p>
+          Stream logs, send signals, and interact with the filesystem of
+          scheduled applications. These operator-friendly commands bring the
+          familiar debugging tools to a scheduled world.
+        </p>
       </div>
     </div>
   </div>
-</div>
+</section>
 
-<div id="subhero-banner" class="banner gray-banner">
+<section id="hybrid-multi-cloud" class="marketing">
   <div class="container">
     <div class="row">
-      <div class="col col-md-12">
-        <p>Nomad scheduled 1 million containers on 5,000 hosts in under 5 minutes</p>
-        <a class="button inline" href="https://www.hashicorp.com/c1m.html">Read more</a>
+      <div class="col-sm-12">
+        <span class="callout">Combine</span>
+        <h2>Hybrid &amp; Multi Cloud</h2>
+        <p class="lead">
+          As more organizations migrate to cloud technologies, the desire to
+          run in multiple datacenters and multiple regions becomes critical.
+          Nomad can span public and private clouds and treat all your
+          infrastructure as a pool of resources.
+        </p>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-sm-12">
+        <%= inline_svg "feature-public-private-clouds.svg" %>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-sm-6">
+        <h3>Hybrid Cloud</h3>
+        <p>
+          Nomad can bridge the gap between the public and private cloud,
+          creating a unified interface for developers to deploy and run any
+          application on any platform. Easily run services across your internal
+          private cloud and external public cloud transparently to application
+          developers.
+        </p>
+      </div>
+      <div class="col-sm-6">
+        <h3>Multi Cloud</h3>
+        <p>
+          As more organizations migrate to cloud technologies, the desire to run
+          applications simultaneously across multiple clouds increases. Nomad
+          combines the power of all cloud providers into a single, unified
+          interface for deploying applications and running services.
+        </p>
       </div>
     </div>
   </div>
-</div>
+</section>
 
-<div id="content">
+<section id="simplify" class="marketing green">
   <div class="container">
     <div class="row">
-      <div class=" col-md-12">
-        <h2 class="feature-section-title">Overview</h2>
+      <div class="col-sm-12">
+        <span class="callout">Maintain</span>
+        <h2>Simplify Operations</h2>
+        <p class="lead">
+          Nomad simplifies operations by supporting blue/green deployments,
```

Comment:
by supporting "blue/green deployments" -> "several deployment strategies to safely upgrade jobs"